cmd/crl-updater/main.go

```diff
@@ -36,13 +37,30 @@ type Config struct {
 		// in CCADB MUST be updated.
 		NumShards int
 
+		// ShardWidth is the amount of time (width on a timeline) that a single
+		// shard should cover. Ideally, NumShards*ShardWidth should be an amount of
+		// time noticeably larger than the current longest certificate lifetime,
+		// but the updater will continue to work if this is not the case (albeit
+		// with more confusing mappings of serials to shards).
+		// WARNING: When this number is changed, revocation entries will move
```

Comment:
Probably worth ensuring that SRE keeps a comment to the same effect next to this key in their repository.

```suggestion
		// WARNING: When this number is changed, revocation entries will move
```